terraform-infra-approvals/global.json

```diff
@@ -31,6 +32,7 @@
     {"source":  "git@github.com:hmcts/cnp-module-action-group?ref=master"},
     {"source":  "git@github.com:hmcts/cnp-module-application-insights?ref=master"},
     {"source":  "git@github.com:hmcts/cnp-module-app-service-plan?ref=master"},
-    {"source":  "git@github.com:hmcts/cnp-module-metric-alert?ref=master"}
+    {"source":  "git@github.com:hmcts/cnp-module-metric-alert?ref=master"},
+    {"source":  "./infrastructure/modules/kv_secrets"}
```

Comment:
repo specific configuration shouldn't be in this file, you should create a file for your repo, see docs: https://github.com/hmcts/sds-jenkins-config/tree/master/terraform-infra-approvals

Reply:
sorry for the mis-reading. sorted now